.tmp.config.yml

```diff
@@ -10,7 +11,9 @@ sendgrid:
   url: "url"
   api_key: "api_key"
   templates:
-    title: template_id
+    otp: template_id
```

Comment:
templates can be static on config no need to read from yaml

Reply:
You mean just an example title:id would be enough?

Reply:
no I mean that you may not use confing.yml at all and can be static